### ISSUE TITLE
'¬' is shown at the end of the last line although there is no '\n'

### ISSUE THREAD

**Issue by timjb** [coderaiser/cm-show-invisibles#12]
The last line does not have a \n at the end because otherwise it wouldn't be the last line.

**Comment by timjb** [comment]
I've found a workaround: Adding the CSS rule

```css
.CodeMirror-code > div:last-child > pre > span::after {
  display: none;
}
```

prevents the ¬ after the last line from rendering.

**Comment by coderaiser** [comment]
Landed in [v3.1.0](https://github.com/coderaiser/cm-show-invisibles/releases/tag/v3.1.0) :tada:

Is it works for you?

**Comment by timjb** [comment]
It works! Thanks a lot :+1:
